rpc-client-api/src/request.rs

```diff
@@ -243,8 +243,7 @@ impl fmt::Display for RpcResponseErrorData {
                 if logs.is_empty() {
                     Ok(())
                 } else {
-                    // Give the user a hint that there is more useful logging information available...
-                    write!(f, "[{} log messages]", logs.len())
+                    write!(f, "[{} log messages: {logs:?}]", logs.len())
```

Comment:
```suggestion
                    writeln!(f, "{} log messages:", logs.len())?;
                    for log in logs {
                        writeln!(f, "  {log:?}")?;
                    }
```
At the risk of prompting someone to ask for `--output json` error logs, what do you think about something like this?

Reply:
That is pretty nice, here's what it looks like:

```
Error: Deploying program failed: RPC response error -32002: Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction 7 log messages:
  Program 11111111111111111111111111111111 invoke [1]
  Program 11111111111111111111111111111111 success
  Program BPFLoaderUpgradeab1e11111111111111111111111 invoke [1]
  Program 11111111111111111111111111111111 invoke [2]
  Program 11111111111111111111111111111111 success
  Unresolved symbol (sol_blake3) at instruction #53 (ELF file offset 0x1a8)
  Program BPFLoaderUpgradeab1e11111111111111111111111 failed: invalid account data for instruction
```

Will do!